pym/bob/cmds/build.py

```diff
@@ -1074,7 +1076,7 @@ def doProject(argv, bobRoot):
         extra.append('-c')
         extra.append(c)
     for e in args.white_list:
-        extra.append('e')
+        extra.append('-e')
```

Comment:
Would you mind splitting out this fix into a separate commit? Fixes and features should not be mixed.